docs/sources/fundamentals/intro-to-prometheus/index.md

```diff
@@ -0,0 +1,110 @@
+
+### Example
+
+The following Grafana dashboard image shows a disk IO graph of raw data from Prometheus that is derived from a laptop.
+
+The **Metrics browser** field contains the following query:
+
+`node_disk_written_bytes_total{job=”integrations/macos-node”, device!=””}`
+
+In this example, the Y-axis shows the total number of bytes written, and the X-axis shows dates and times. As the laptop runs, the number of bytes written increases over time. Below **Metrics browser** is a counter that counts the number of bytes written over time.
+
+{{< figure src="/media/docs/grafana/intro-prometheus/dashboard-example.png" max-width="750px" caption="Metrics browser and counter" >}}
+
+The query is a simple example of [PromQL](/blog/2020/02/04/introduction-to-promql-the-prometheus-query-language/), the Prometheus Query Language. It identifies the metric of interest (`node_disk_written_bytes_total`) and provides two labels (`job` and `device`). The label selector `job=”integrations/macos-node”` is used to filter metrics. It reduces the scope of the metrics to those coming from the MacOS integration job, and specifies that the “device” label cannot be empty. The result of this query is the raw stream of numbers that the graph shows.
+
+Although this view provides some insight into the performance of the system, it doesn’t provide the full story. For a clearer picture of system performance, it is also important to understand the rate of change that shows _how fast the data being written is changing_. To properly monitor disk performance, it is important to see spikes in activity to understand if and when the system is under load, and whether or not disk performance is at risk. PromQL includes a [rate()](https://prometheus.io/docs/prometheus/latest/querying/functions/#rate) function that shows the per-second average rate of increase over `5m` (5-minute) intervals. This view provides a much better idea of what’s happening with the system.
+
+{{< figure src="/media/docs/grafana/intro-prometheus/rate-function.png" max-width="750px" caption="Prometheus rate function" >}}
+
+A counter metric is just one type of metric. Prometheus [supports several others](https://prometheus.io/docs/concepts/metric_types/). A counter is a number (such as total bytes written) that only goes up. A second metric type is called a `gauge`, which can go up or down.
+
+The following gauge visualization shows the total RAM usage on a computer.
+
+{{< figure src="/media/docs/grafana/intro-prometheus/gauge-example.png" max-width="750px" caption="Gauge visualization" >}}
+
+The third metric type is called a `histogram`, which counts observations and puts them into configurable groups. The following example shows floating-point numbers grouped into ranges that show how often each occurred.
```

Comment:
idk if it matters, but I think there is also the `summary` type of metric

Reply:
@moxious , would it make sense to add summary?